### PR TITLE
test: sqlness test for alter table rename

### PR DESCRIPTION
**Pull request by waynexia**
Signed-off-by: Ruihang Xia <waynestxia@gmail.com>

I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Sqlness test cases for `ALTER TABLE RENAME`.

Current behavior doesn't work as expected. As described in #892

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: tests/cases/standalone/alter/rename_table.result [hunk 1]
```diff
@@ -0,0 +1,68 @@
+CREATE TABLE t(i INTEGER, j BIGINT TIME INDEX);
+
+Affected Rows: 0
+
+SHOW CREATE TABLE t;
```
Comment (killme2008) [inline tests/cases/standalone/alter/rename_table.result:5]: Use `desc table [table]` instead?

Comment (waynexia) [reply tests/cases/standalone/alter/rename_table.result]: nice suggestion 👍

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by evenyag — APPROVED** [review]

**Review by evenyag — APPROVED** [review]

**Comment by e1ijah1** [comment]
Is it possible to run SQLness Test in cluster mode now?

**Comment by MichaelScofield** [comment]
@e1ijah1 Possible, but currently not implemented. I've planned to do it, but you can give it a shot~

**Comment by waynexia** [comment]
> Is it possible to run SQLness Test in cluster mode now?

@e1ijah1 `sqlness-runner` has an unimplemented `start_distributed` method which is the entry to start a distributed cluster for testing. You can start from here.

**Pull request merged**